webcomponents/tool/src/main/frontend/packages/sakai-rubrics/src/SakaiRubric.js

```diff
@@ -69,7 +69,7 @@ export class SakaiRubric extends RubricsElement {
             rubricSite.toLowerCase().includes(search);
   }
 
-  shouldUpdate() { return this.rubric; }
+  shouldUpdate() { return super.shouldUpdate() && this.rubric; }
```

Comment:
Just curious, what's the reason for this addition here?

Reply:
Or anything else the base class needs to render